### DISCUSSION TITLE
How to change and assign hotkeys for a noob, please?

### DISCUSSION THREAD

**Discussion by truevectortrue** [jgraph/drawio#4182] [Q&A]
I would be so grateful if someone provides me with a valuable information about how to change hotkeys in this wonderful programm, if possible. I just want to change mouse wheel to zoom-in-out without pressing cntrl also, I don't need the screen to be scrolled up and down instead.
Thank you in advance.

**Comment by PygmaliAnon** [comment]
Same here, same use case, and having a way to map custom keyboard shortcuts to specific actions, like specific shapes and so on would be super helpful.

**Comment by lemerecode** [comment]
I have same issue. The fact that it cant zoom on scroll drives me insane!
I tried to fiddle with a code a bit, i cant get it work. 
Can someone from dev team answer? Is there a technical obstacle? Please answer. Please please please!

**Comment by alderg** [comment]
Try Ctrl+Scrollwheel

**Comment by Ig0rFB** [comment]
Is there still no way to easily remap shortcuts?
